### ISSUE TITLE
Port old ToolingAPIDemo.cls to new SOAP Edition

### ISSUE THREAD

**Issue by afawcett** [afawcett/apex-toolingapi#28]
We need to port the old examples class to the new style SOAP edition version.

**Comment by afawcett** [comment]
@dancinllama Just looking at the Symbol Table types, for example the Method class, it doesn't appear to extend anything, i'm pretty sure last i looked at the WSDL it should? Did the new WSDL2Apex tool miss this one? I wanted to start to port the ToolingAPIDemo.cls over since the new version doesn't have this yet.

**Comment by dancinllama** [comment]
Don't believe that's the case.  There's nothing in the WSDL that extends SymbolTable.  All of the class types (ApexClassMember, ApexTriggerMember, etc) have a SymbolTable member variable, but they're all the same object, which I think is correct.  I could be wrong though, as it's been awhile since I dug  into the SymbolTable.

**Comment by afawcett** [comment]
Yeah looks like a bit more tweaking is required for the SOAP types, if you look at [Method](https://github.com/afawcett/apex-toolingapi/blob/apex-toolingapi-rest/src/classes/ToolingAPI.cls#L1159) from the REST wrapper and in the WSDL, it extends Constructor.

~~~~
<xsd:complexType name="Method">
<xsd:complexContent>
<xsd:extension base="tns:Constructor">
<xsd:sequence>
<xsd:element name="returnType" type="xsd:string"/>
</xsd:sequence>
</xsd:extension>
</xsd:complexContent>
</xsd:complexType>
~~~~
